extensions/extensions-helm/aissemble-infrastructure-chart/values.template.yaml

```diff
@@ -12,7 +12,23 @@ argo-cd:
   global:
     domain: argocd.localdev.me
   configs:
-    params: 
+    cm:
+      resource.customizations: |
+        argoproj.io/Application:
+          health.lua: |
+            hs = {}
+            hs.status = "Progressing"
+            hs.message = ""
+            if obj.status ~= nil then
+              if obj.status.health ~= nil then
+                hs.status = obj.status.health.status
+                if obj.status.health.message ~= nil then
+                  hs.message = obj.status.health.message
+                end
+              end
+            end
+            return hs
```

Comment:
I: this is to ensure the sync wave works among apps. ref: https://argo-cd.readthedocs.io/en/stable/operator-manual/upgrading/1.7-1.8/#health-assessement-of-argoprojioapplication-crd-has-been-removed

Reply:
Might be worth adding this as a comment in the values file.